test/broadway_test.exs

```diff
@@ -487,6 +481,69 @@ defmodule BroadwayTest do
       assert named_priority(get_processor(broadway_name, :default)) == {:priority, :high}
     end
 
+    test "supports starting under a via tuple" do
+      defmodule MyRegistry do
+        def start_link do
+          Registry.start_link(keys: :unique, name: __MODULE__)
+        end
+
+        def via_tuple(name) do
+          {:via, Registry, {__MODULE__, name}}
+        end
+
+        def whereis_name(name) do
+          Registry.whereis_name({__MODULE__, name})
+        end
+      end
+
+      {:ok, _registry} = MyRegistry.start_link()
+
+      defmodule SomeBroadwayNoProcessName do
+        use Broadway
+
+        def handle_message(_, _, _), do: nil
+        def handle_batch(_, _, _, _), do: nil
+      end
```

Comment:
I believe we can instead use any of the other Broadway pipelines in this module. :)